RELEASE_NOTES.md

```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
 
 This debugging feature will mostly be useful to people who are already Ruby experts.
 
+### New ksh resource
 
+ksh is now available to use as a resource and is available as an interpreter in the script resource.
+
+Please see the DOC_CHANGES for more details.
+
+## Better handling of Windows service logging
+
+Currently, the windows service manager passes in a default log_location to the Chef windows service at startup. Once the service is installed, this log_location overrides any log_location set in the client.rb. In 12.6.0, this behavior is changed to allow the Chef windows service to prefer the log_location in client.rb instead. Now, the windows service manager will not explicitly pass in a log_location, and therefore the Chef service will always use what is in the client.rb or the typical default path if none is configured. This enables scenarios such as logging to the Windows event log when running chef client as a Windows service.
+
+## Dsc_resource changes and fixes
+
+* A fix was made for the Nov 2015 update of Windows 10, where the dsc_resource did not properly show the command output when converging the resource.
+* Dsc_resource could in some cases show the plaintext password when #inspected - this is now prevented from happening.
+* Previously, Chef required the LCM Refreshmode to be set to Disabled when utilizing dsc_resource. Microsoft has relaxed this requirement in WFM 5 (PowerShell 5.0.10586.0 or later).  Now, we only require the RefreshMode to be disabled when running on earlier versions of PowerShell.
```

Comment:
I'd call out the full name - Windows Management Framework 5 (WMF5).  And you've got the F and M swapped.

Also, I'd say for the last sentence on this line - 
Now, we only require the RefreshMode to be disabled when running on earlier versions of PowerShell 5.